Makefile

```diff
@@ -0,0 +1,11 @@
+PUSH_DOCKER_IMAGE := true
+SUFFIX := $(shell echo $$PLATFORM | sed 's/\//-/' | sed 's/\///')
+
+.PHONY: build-push-docker-image
+build-push-docker-images:
+	@DOCKER_BUILDKIT=1 docker buildx build \
+		--platform ${PLATFORM} \
+		--output "type=image,push=${PUSH_DOCKER_IMAGE}" \
+		--build-arg ARCH="${ARCH}" \
+		-f ampd/Dockerfile \
+		-t axelarnet/axelar-ampd-${SUFFIX}:${SEMVER} --provenance=false .
```

Comment:
whats the flag for ?

Reply:
The provenance flag in Docker buildx generates detailed records about a Docker image's build process. Build timestamps, parameters, and source code etc...

We added it because it was a breaking change done on buildx.
It was a workaround we found on forums, because other people faced the same issue when building mutliarch images.
More context here : https://interoplabs.slack.com/archives/C01RT8WDK1B/p1674502980935789?thread_ts=1674499987.346189&cid=C01RT8WDK1B